pygitops/operations.py

```diff
@@ -67,7 +69,9 @@ def stage_commit_push_changes(
 
     # push changes to the remote branch
     origin = repo.remotes.origin
-    push_info = origin.push(branch_name)[0]
+    if not kwargs_to_push:
+        kwargs_to_push = {}
+    push_info = origin.push(branch_name, **kwargs_to_push)[0]
```

Comment:
```suggestion
    push_info = origin.push(branch_name, **kwargs_to_push or {})[0]
```

this removes the need for the above conditional